src/app/src/components/AboutModal.jsx

```diff
@@ -40,49 +40,32 @@ const AboutModal = ({ isOpen, onClose }) => {
+                                over 200 countries. Public, aggregate datasets
+                                for the 2020 and 2021 surveys at the{' '}
                                 <StyledLink
-                                    href='https://sdgs.un.org/goals'
-                                    title='SDGs'
+                                    href='https://data.humdata.org/dataset/survey-on-gender-equality-at-home'
+                                    title='Humanitarian Data Exchange website'
                                 />
-                                ). In 2020, the COVID-19 pandemic provided a
-                                special impetus to examine these issues through
-                                the survey.
-                            </StyledText>{' '}
-                            <StyledText>
-                                The survey measures attitudes and beliefs about
-                                gender equality, participation in unpaid care
-                                and domestic work, resource allocation, and
-                                COVID-19’s impact on everyday lives. The data
-                                sheds light on differences within countries and
-                                among world regions regarding the extent to
-                                which people agree that men and women should
-                                have equal opportunities. The data also
-                                highlights gender and region-specific challenges
-                                of life during the pandemic, such as food
-                                insecurity, access to education and healthcare,
-                                and employment prospects.
-                            </StyledText>
-                            <StyledText>
-                                Contact{' '}
-                                <StyledLink
-                                    href='mailto:gendersurvey@fb.com'
-                                    title='gendersurvey@fb.com'
-                                />{' '}
-                                with any questions.
+                                , and de-identified, record-level response data
+                                to most questions may be available to research
+                                and nonprofit organizations for research
+                                purposes. Meta only shares data publicly once it
+                                is aggregated to the country or region level.
+                                Before sharing with Data for Good partners, Meta
+                                ensures that all survey data are de-identified.
                             </StyledText>
                         </StyledAccordionItem>
                         <StyledAccordionItem title='How the survey contributes to Facebook’s mission and the UN Sustainable Development Goals'>
```

Comment:
Should Facebook be switched to Meta here?

Reply:
Confirmed with Samhir